code/mob/input.dm

```diff
@@ -202,6 +202,12 @@
 						last_move_trigger = ticker ? ticker.round_elapsed_ticks : 0 //Wire note: Fix for Cannot read null.round_elapsed_ticks
 						deliver_move_trigger(running ? "sprint" : m_intent)
 
+					// Tripping (the physical kind)
+					var/trip_chance = 5
+					if (src.traitHolder && src.traitHolder.hasTrait("trippy") && prob(trip_chance))
+						src.setStatus("resting", INFINITE_STATUS)
+						src.force_laydown_standup()
+						src.visible_message(SPAN_ALERT("<B>[src]</B> trips!"))
```

Comment:
isn't there a "trip" emote (for humans at least) already? Might be worth considering

Reply:
Yeah, but that one force paralyses for 3 seconds and is pretty much only used for `foreign_limb_effect()`, seemed excessive.